docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1095 @@
+     "id_code": {
+       "vat_number": "IT98765432101",
+       "legal_identifier": "98765432101"
+     },
+     "organization_name": "Badge Services Ltd",
+     "email": "compliance@rprivate-badge.ci.example.com",
+     "authorized_claims": ["given_name", "family_name", "company_id"],
+     "authorized_credential_types": ["example-company-badge"],
+     "scope_restrictions": {
+       "domains": ["MEMBERSHIP"],
+       "purposes": ["ASSOCIATION"]
+     }
+   }
+
+
+
+Federation Entities MUST integrate Trust Marks in their Entity Configuration using the ``trust_marks`` claim as specified in :ref:`trust:Entity Configuration Leaves and Intermediates`. Entities MAY receive multiple Trust Marks for different authorization scopes.
+
+
+.. code-block:: json
+
+   {
+     "iss": "https://credentials.example.gov",
+     "sub": "https://credentials.example.gov",
+     "jwks": { },
+     "authority_hints": ["https://trust-anchor.eid-wallet.example.it"],
+     "trust_marks": [
+       {
+         "trust_mark_type": "https://trust-anchor.eid-wallet.example.it/trust_marks/federation-entity/credential-issuer",
+         "trust_mark": "eyJhbGciOiJFUzI1NiIsImtpZCI6IlRydXN0QW5jaG9yS2V5SWQiLCJ0eXAiOiJKV1QifQ..."
+       }
+     ],
+     "metadata": { }
+   }
+
+
+.. code-block:: json
+
+   {
+     "iss": "https://healthcare-ci.example.gov",
+     "sub": "https://healthcare-ci.example.gov",
+     "jwks": { },
+     "authority_hints": ["https://healthcare.intermediate.eid-wallet.example.it"],
+     "trust_marks": [
+       {
+         "trust_mark_type": "https://healthcare.intermediate.eid-wallet.example.it/trust_marks/federation-entity/credential-issuer",
+         "trust_mark": "eyJhbGciOiJFUzI1NiIsImtpZCI6IkhlYWx0aGNhcmVJbnRlcm1lZGlhdGVLZXlJZCIsInR5cCI6IkpXVCJ9..."
+       }
+     ],
+     "metadata": { }
+   }
+
+
+Trust Mark Validation
+"""""""""""""""""""""
+
+Federation participants validate Trust Mark status through two mechanisms:
+
+1. **Static Validation**: Cryptographic verification using the issuing Federation Authority's public key from the trust chain.
+2. **Dynamic Validation**: Real-time status verification via the issuing Federation Authority's ``/trust_mark_status`` endpoint as defined in :ref:`trust:Federation API endpoints`.
+
+
+Certificate Management Operations
+----------------------------------
+
+This section defines the operational procedures for X.509 certificate management within the IT-Wallet federation, covering certificate chain analysis, validation procedures, and revocation verification. These procedures complement the federation onboarding processes and support ongoing certificate lifecycle management for all federation participants.
+
+Federation PKI Architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The IT-Wallet federation operates a hierarchical Public Key Infrastructure where:
+
+	- **Trust Anchor**: Acts as Root Certificate Authority where root certificates MUST NOT exceed **5-year validity period**.
```

Comment:
```suggestion
	- **Trust Anchor**: Acts as Root Certificate Authority where root certificates MUST NOT exceed **5-year validity period**.
```

5 probably is too much

Reply:
This is the root CA. When you say that it is "too much", are you basing it on some known standard or authorative reference? As far as I know, a root CA is typically valid for 10-20 years. we should have a kind of hierarchy in the validity period (root CA, subCAs, leaves certificates).